### DISCUSSION TITLE
在OBS中不使用绿幕完成自动抠人像

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#9] [General]
# 在OBS中不使用绿幕完成自动抠人像

在OBS中不使用绿幕完成自动抠人像 - 来自Xzonn的小站

https://xzonn.top/posts/Automate-Portrait-Keying-in-OBS-without-Green-Screen.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Psykun 评论于 2022-12-07 18:09:18 (UTC+8)*****

卧槽，太机智了！ :grin:

**Comment by sailorsale** [comment]
窗口绿幕用滤镜色值去除后，边缘残留能否解决

**Reply by Xzonn** [reply]
可以试试调整 OBS 里的滤镜参数，改变平滑值；或者让背景尽可能地不要有太多杂色，腾讯会议的虚拟背景毕竟还是靠实时分析图像，背景简单也有利于软件运行。
